src/yunohost/__init__.py

```diff
@@ -53,6 +53,7 @@ def check_command_is_valid_before_postinstall(args):
 
     allowed_if_not_postinstalled = ['tools postinstall',
                                     'tools versions',
+                                    'tools shell',
```

Comment:
Is it usable from the API ?
The risk is we could forget this, the day we secure the access to `tools postinstall` from the API...

Reply:
No it's not, the only route exposed to unauthenticated users is the postinstall : https://github.com/YunoHost/yunohost/blob/dev/data/actionsmap/yunohost.yml#L1383-L1384

(then the postinstall is ran and /etc/yunohost/installed is created so it can't be re-run anymore)

Reply:
Ok, so, it's solved